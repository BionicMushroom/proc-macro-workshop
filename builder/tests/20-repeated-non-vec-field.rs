use derive_builder::Builder;

#[derive(Builder)]
struct Test {
    #[builder(each = "whatever")]
    field: std::result::Result<u32, u32>,
}

fn main() {}
