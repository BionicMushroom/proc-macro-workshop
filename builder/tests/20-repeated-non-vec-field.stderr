error: expected standard Vec
 --> tests/20-repeated-non-vec-field.rs:6:12
  |
6 |     field: std::result::Result<u32, u32>,
  |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
